ssz/sedes/vector.py

```diff
@@ -28,9 +28,10 @@ class Vector(
     ]
 ):
     def __init__(self, element_sedes: TSedes, length: int) -> None:
-        if length <= 0:
+        if length < 1:
```

Comment:
Why the change here? `<= 0` is equivalent to `< 1`

Reply:
Yes, it's equivalent. I wanted to make it more clear and follow the error message more tightly.